feather/base/src/anvil/entity.rs

```diff
@@ -200,7 +189,7 @@ impl From<&ItemData> for ItemStack {
     fn from(item: &ItemData) -> Self {
         ItemNbt::item_stack(
             &item.nbt,
-            Item::from_name(item.item.as_str()).unwrap_or(Item::Air),
+            Item::from_name(item.item.as_str()).unwrap(),
```

Comment:
This would cause a panic on loading an invalid player.dat file. `minecraft-data` removed `Item::Air` and other non-item blocks from items.json, and I don't think panicking on invalid world saves is bad. Anyway, it's better than just silently wiping out a player's inventory.

Reply:
Would it be better to only implement `TryFrom`?